src/sdk/models/policies.ts

```diff
@@ -86,14 +86,18 @@ export interface PolicyTrackingGroupRule extends IEntityIdentifier<string> {
 export interface PolicyActionDescriptor extends IEntityIdentifier<string> {
 	name?: string;
 	description?: string;
+	destructive?: boolean;
```

Comment:
If we put these properties here, then `PolicyTrackingGroupActionConfiguration` can't extend this interface, because it doesn't have any of these.
Might be best to do:
```ts
interface PolicyActionBase extends IEntityIdentifier<string> {
	name?: string;
	description?: string;
}

export interface PolicyActionDescriptor extends PolicyActionBase {
	destructive?: boolean;
	exclusive?: boolean;
	requires_existing?: boolean;
	removes_item?: boolean;
	prompt?: any;
}

export interface PolicyTrackingGroupActionConfiguration extends PolicyActionBase {
	[name: string]: any;
}
```

Reply:
Updated in new commit 21eb0e4.